<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestGUI</class>
 <widget class="QMainWindow" name="TestGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1201</width>
    <height>849</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Test GUI</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="testLabel">
        <property name="text">
         <string>Test Name</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTabWidget" name="tab_5">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab_4">
         <attribute name="title">
          <string>Test</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QFormLayout" name="formLayout">
              <item row="1" column="0">
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Object Type</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QComboBox" name="objectTypeCoB"/>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Name</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="testNameLE"/>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Version</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <layout class="QFormLayout" name="formLayout_2">
                <item row="0" column="0">
                 <widget class="QLabel" name="label_4">
                  <property name="text">
                   <string>Major</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="label_5">
                  <property name="text">
                   <string>Minor</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QSpinBox" name="versionMajorSB"/>
                </item>
                <item row="1" column="1">
                 <widget class="QSpinBox" name="versionMinorSB"/>
                </item>
               </layout>
              </item>
              <item row="4" column="0">
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>Comment</string>
                </property>
               </widget>
              </item>
              <item row="4" column="1">
               <widget class="QTextEdit" name="testCommentTE"/>
              </item>
              <item row="5" column="0">
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>Report Templates</string>
                </property>
               </widget>
              </item>
              <item row="6" column="0">
               <widget class="QLabel" name="label_8">
                <property name="text">
                 <string>Debrief Templates</string>
                </property>
               </widget>
              </item>
              <item row="5" column="1">
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <widget class="QListWidget" name="reportTemplatesListWidget"/>
                </item>
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_4">
                  <item>
                   <widget class="QPushButton" name="addReportTemplatePB">
                    <property name="text">
                     <string>+</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="removeReportTemplatePB">
                    <property name="text">
                     <string>-</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
              <item row="6" column="1">
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QListWidget" name="debriefTemplatesListWidget"/>
                </item>
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_5">
                  <item>
                   <widget class="QPushButton" name="addDebriefTemplatePB">
                    <property name="text">
                     <string>+</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="removeDebriefTemplatePB">
                    <property name="text">
                     <string>-</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="label_9">
                <property name="text">
                 <string>Active</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QCheckBox" name="isActiveChB">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="editPB">
              <property name="text">
               <string>Edit</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="savePB">
              <property name="text">
               <string>Save</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="cancelPB">
              <property name="text">
               <string>Cancel</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Run Description</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="DescriptionEditorWidget" name="descriptionEditorWidget" native="true"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Scoring</string>
         </attribute>
         <layout class="QHBoxLayout" name="scoringTabMainLayout">
          <item>
           <widget class="ScoreTestViewer" name="scoreTestViewer" native="true"/>
          </item>
         </layout>
        </widget>
        <widget class="WinsTabWidget" name="winsDescriptionEditor">
         <attribute name="title">
          <string>Wins</string>
         </attribute>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1201</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionTracksGUI"/>
   </widget>
   <addaction name="menuTools"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionTracksGUI">
   <property name="text">
    <string>Tracks GUI</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DescriptionEditorWidget</class>
   <extends>QWidget</extends>
   <header>main.scripts.gui.test_gui.description_editor</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ScoreTestViewer</class>
   <extends>QWidget</extends>
   <header>main.scripts.gui.test_gui.test_viewer</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WinsTabWidget</class>
   <extends>QWidget</extends>
   <header>main.scripts.gui.test_gui.wins_tab_widget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
